docs/src/getting-started/contracts.md

```diff
@@ -0,0 +1,34 @@
+# Deploying contracts
+
+There are two main ways of working with contracts in the SDK: deploying a contract with SDK or using the SDK to connect to existing contracts.
```

Comment:
"already-deployed" is a mouthful IMO haha. How about just "deployed"? So it would be "using the SDK to interact with deployed contracts." We replace `connect` with `interact`, and it should be inferrable that, by using this, you're only skipping the deployment step.